Prox/Prox/PlaceDetails/PlaceDetailViewController.swift

```diff
@@ -309,6 +311,7 @@ class PlaceDetailViewController: UIViewController {
                         mapButtonBadge.widthAnchor.constraint(greaterThanOrEqualToConstant: 20.0)]
 
         NSLayoutConstraint.activate(constraints, translatesAutoresizingMaskIntoConstraints: false)
+        NSLog("View finished loading")
```

Comment:
leftovers from things that I tried to fix the non-interaction of details cards on display. Will remove